source/js/components/search-form.js

```diff
@@ -0,0 +1,108 @@
+import fetchUtils from "@degjs/fetch-utils";
+
+const searchForm = () => {
+
+
+    const formEl = document.querySelector('.js-search-form');
+    const layoutEl = document.querySelector('.js-general-content');
+    const searchFieldSet = document.querySelector('.js-search-fieldset');
+    let searchInputEl;
+    const searchButtonEl = formEl.querySelector('.js-search-btn');
+
+    const url = formEl.dataset.api;
+
+    function init() {
+        searchInputEl = formEl.querySelector('.js-search');
```

Comment:
[question] Just curious why this is the only querySelector inside of init? Couldn't it also be elevated?

Reply:
the reason I added to the init is because init runs once so the variable elevates once and assigns value to the outer scope var and that outer scope var can be called multiple times without querying the dom.

Reply:
Why would putting the query in the outer scope cause it to query the DOM multiple times?

Reply:
because the variable will have a queryselector method which needs to elevate every time getData calls it.

Reply:
I may not be explaining my question well. What I'm asking is, what makes `searchInputEl` any different from the other elevated variables that have queries in them? https://screencast.com/t/aaLT2qirt

Reply:
Welp! I think those need to inside the init as well.

Reply:
That wasn't quite the takeaway I was hoping for ;) You certainly can do that if you want, but I guess I'm trying to get at whether you think the variables in the outer scope are being created, queried, etc. more than once?

Reply:
sorry about my short answer (I jump into fixing it..lol.), yes those variables are queried more than once as well so by moving it to init we scope the elevation so it can elevate once which will assign values to outer scope variables and outer scope variables can be utilized multiple times.

Reply:
Why would they be queried more than once?